ssg/content_diff.py

```diff
@@ -101,7 +101,7 @@ def _get_list_of_platforms(self, cpe_platforms, idref):
         for cpe_platform in cpe_platforms:
             fact_refs = cpe_platform.find_all_fact_ref_elements()
             for fact_ref in fact_refs:
-                cpe_list.append(fact_ref.get("name"))
+                cpe_list.append(fact_ref.get("id-ref"))
```

Comment:
GH doesn't allow to suggest changes for some reason, but `.*fact_ref.*` should be renamed to `...check_fact_ref...`

Reply:
Done